bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,163 @@
+    createOption("Montana", state);
+    createOption("Nebraska", state);
+    createOption("Nevada", state);
+    createOption("New Hampshire", state);
+    createOption("New Jersey", state);
+    createOption("New Mexico", state);
+    createOption("New York", state);
+    createOption("North Carolina", state);
+    createOption("North Dakota", state);
+    createOption("Ohio", state);
+    createOption("Oregon", state);
+    createOption("Pennsylvania", state);
+    createOption("Rhode Island", state);
+    createOption("South Carolina", state);
+    createOption("South Dakota", state);
+    createOption("Tennessee", state);
+    createOption("Texas", state);
+    createOption("Utah", state);
+    createOption("Vermont", state);
+    createOption("Virginia", state);
+    createOption("Washington", state);
+    createOption("West Virginia", state);
+    createOption("Wisconsin", state);
+    createOption("Wyoming", state);
+
+    stateDiv.appendChild(state);
+    addressInputDiv.appendChild(stateDiv);
+    
+    //Zip Code
+    createTextInputElement("Postal", document.getElementById("address-input-div"));
+}
+
+function loadCanadianAddress() {
+    //Address line
+    let addressInputDiv = document.getElementById("address-input-div");
+
+    createTextInputElement("Address Line 1", document.getElementById("address-input-div"));
+    createTextInputElement("Address Line 2", document.getElementById("address-input-div"));
+
+    //City
+    createTextInputElement("City", document.getElementById("address-input-div"));
+
+    //Province
+    let provinceDiv = document.createElement("div");
+    let province = document.createElement("Select"); 
+    province.setAttribute("id", "province-input"); 
+    province.setAttribute("name", "province-input"); 
+    let provinceLabel = document.createElement('label');
+    provinceLabel.setAttribute("for", "state-input");
+    let provinceLabelText = document.createTextNode("Province: ");
+    provinceLabel.appendChild(provinceLabelText);
+    provinceDiv.appendChild(provinceLabel);
+
+    createOption("Alberta", province);
+    createOption("British Columbia", province);
+    createOption("Manitoba", province);
+    createOption("Alberta", province);
+    createOption("New Brunswick", province);
+    createOption("Newfoundland and Labrador", province);
+    createOption("Northwest Territories", province);
+    createOption("Nova Scotia", province);
+    createOption("Nunavut", province);
+    createOption("Ontario", province);
+    createOption("Prince Edward Island", province);
+    createOption("Quebec", province);
+    createOption("Saskatchewan", province);
+    createOption("Yukon", province);
+
+    provinceDiv.appendChild(province);
+    addressInputDiv.appendChild(provinceDiv);
+
+    //Postal Code
+    createTextInputElement("Postal", document.getElementById("address-input-div"));
+    
+}
+
+function createOption(optionValue, selectToAppend) {
+    let option = document.createElement("option");
```

Comment:
You should be able to use const instead of let here =D

Reply:
Made const